pywbemtools/pywbemcli/_cmd_class.py

```diff
@@ -1054,20 +1054,24 @@ def get_namespaces(context, namespaces, default_rtn=False):
 
       context context (:class:`ContextObj` provided with command)
 
-      namespaces (list of (:term: str)
+      namespaces (tuple  or :term:string)
```

Comment:
Was the change from `of` to `or` intentional?

Reply:
It was booboo should  be tuple of :term:string or :term:string. Fixed.  Tuple because that is what click presents.

Reply:
DONE